### DISCUSSION TITLE
About docker deployment

### DISCUSSION THREAD

**Discussion by xianyuhuang** [fluent/fluent-bit#9340] [Q&A]
I'm wondering how I'm supposed to mount the config file after I pull the image from the docker container

**Comment by patrick-stephens** [comment]
The same way you do any mounting in docker, you can either mount to the default location `/fluent-bit/etc/fluent-bit.conf` or mount somewhere else then pass it as an argument to `-c <filename>`

I have some examples here: https://chronosphere.io/learn/fluent-bit-tips-tricks/

e.g.
```shell
docker run --rm -it -v $PWD/fluent-bit.conf:/fluent-bit/etc/fluent-bit.conf:ro fluent/fluent-bit
```

or 

```shell
docker run --rm -it -v $PWD/fluent-bit.conf:/some/other/location.conf:ro fluent/fluent-bit -c /some/other/location.conf
```
